packages/api/server.js

```diff
@@ -14,11 +14,43 @@
 // limitations under the License.
 
 const express = require('express');
+const Repository = require('./src/repository');
 const app = express();
 const bodyParser = require('body-parser');
 
 app.use(bodyParser.json());
 
+app.get('/repos', async (req, res) => {
+  // TODO: Make it return more information about the repos, beyond just their names
+  var repository = new Repository(null);
+  var result = await repository.getCollection('dummy-repositories');
+
+  let repoNames = [];
+
+  for (var index = 0; index < result.length; index++) {
```

Comment:
nit: `let` rather than `var`...

Here's a blog post [on the topic](https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75) basically, you want to opt for `let` and `const` rather than `var` these days:

* `let`: if you need to reassign.
* `const`: if you do not.